### ISSUE TITLE
ngx-logger 4.3.1 not compiling on ng serve (Angular v12)

### ISSUE THREAD

**Issue by StefDardennePXL** [dbfannin/ngx-logger#259]
Hi community,

Thank you for publishing the package ngx-logger, it helps us a lot.

After installing the package by following the steps in the docs, my application wouldn't serve anymore due to missing dependencies on the entrypoint 'ngx-logger'. I already tried reinstalling my node_modules (also removed package-lock.json) and cleared the npm cache.

v4.2.2 seems to work just fine.

v4.3.1 throws an error in regard to the target entry-point @angular/...
I verified that @angular is installed.

Thank you for helping me addressing this issue.

![Knipsel](https://user-images.githubusercontent.com/43743826/125413193-ea0e5a80-e3bf-11eb-94ed-5002f5e2c861.PNG)

**Comment by SnakeSVx** [comment]
Same issue after upgrading to the latest version 4.3.1.

Cannot find module 'ngx-logger' or its corresponding type declarations.

1 import { NGXLoggerMonitor, NGXLogInterface } from 'ngx-logger';
                                                    ~~~~~~~~~~~~
ngx-logger can be found in node_modules and seems to be installed correctly. But the compilers doesn't seem to recognise it.

**Comment by vinayjangir7** [comment]
I am facing a similar issue.

**Comment by carlos-ch** [comment]
Same issue!
<img width="1108" alt="Screenshot 2021-07-13 at 13 56 17" src="https://user-images.githubusercontent.com/61248286/125447540-3d3689b5-c1a3-406c-8eba-aca433cbccb2.png">

**Comment by dbfannin** [comment]
Good morning. It looks like there was an issue when publishing the library with 4.3.1. This should be resolved in v4.3.2 Sorry for the inconvenience!
 
@StefDardennePXL - If you don't mind closing out this ticket once you've verified 4.3.2 works for you. Thanks!
